app/controllers/shop_product_sink/webhooks_controller.rb

```diff
@@ -41,6 +41,8 @@ def resource_class
     end
 
     def shopify_resource
+    	# set site to string or ActiveResource raises on call to .prefix on nil
+    	ShopifyAPI::Base.site = '' if ShopifyAPI::Base.site.nil?
```

Comment:
Nope. You got me here. I can't reproduce that ActiveResource exception anymore. It looks related to authenticating as a private app, but that's just a guess. What's the best way to go about reverting this change? New commit and an extra PR?